### PR TITLE
Enable Back-Channel logout by default

### PR DESCRIPTION
**Pull request by Kayathiri-Mahendrakumaran**

### CODE REVIEW

File: sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/bcIFrame.jsp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<html>
+<head>
+    <title>Back channel Logout Session Management RP IFrame</title>
```
Comment (ashendes) [inline sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/bcIFrame.jsp:3]: `Backchannel` should be one word. Please update the title.

File: sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/bcIFrame.jsp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <!-- JQuery -->
+    <script src="libs/jquery_3.3.1/jquery.min.js"></script>
+    <script language="JavaScript" type="text/javascript">
+
+    function check_session() {
+        $.ajax({
+            url:'checkBclogout',
```
Comment (ashendes) [inline sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/bcIFrame.jsp:10]: Lets change the URL path to kebab case.

File: sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/home.jsp [hunk 3]
```diff
@@ -374,6 +374,17 @@
 <%
     }
 %>
+<%
+    boolean enableOIDCBackchannelLogout = true;
+    if (session.getAttribute(OAuth2Constants.OIDC_BACK_CHANNEL_LOGOUT_ENABLED) != null){
+        enableOIDCBackchannelLogout = (boolean)session.getAttribute(OAuth2Constants.OIDC_BACK_CHANNEL_LOGOUT_ENABLED);
+    }
+    if(enableOIDCBackchannelLogout){
```
Comment (ashendes) [inline sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/home.jsp:382]: Please fix the formatting issues.

File: sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/bcIFrame.jsp (hunk 1 above)
Comment (Kayathiri-Mahendrakumaran) [reply sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/bcIFrame.jsp]: Addressed with 5a0afb6d740f15ba4e8473e191aea05c4c08dc23

File: sso-samples/oidc-sso-sample/pickup-dispatch/src/main/org/wso2/sample/identity/oauth2/logout/LogoutCheckServlet.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+/**
+ * Servlet for checking Backchannel logout.
+ */
+public class LogoutCheckServlet extends HttpServlet {
```
[the remark this answers is not in the document]
Comment (Kayathiri-Mahendrakumaran) [reply sso-samples/oidc-sso-sample/pickup-dispatch/src/main/org/wso2/sample/identity/oauth2/logout/LogoutCheckServlet.java]: Addressed with 5a0afb6d740f15ba4e8473e191aea05c4c08dc23

File: sso-samples/oidc-sso-sample/pickup-manager/pom.xml [hunk 5]
```diff
@@ -66,6 +66,10 @@
             <artifactId>claim-manager</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
```
Comment (ashendes) [inline sso-samples/oidc-sso-sample/pickup-manager/pom.xml:71]: This seems to be a redundant dependency. Please check

**Review by ashendes — APPROVED** [review]

File: sso-samples/oidc-sso-sample/pickup-manager/src/main/webapp/bcIFrame.jsp [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<html>
+<head>
+    <title>Backchannel Logout Session Management RP IFrame</title>
+    <!-- JQuery -->
+    <script src="libs/jquery_3.3.1/jquery.min.js"></script>
+    <script language="JavaScript" type="text/javascript">
+
+    function check_session() {
+        $.ajax({
+            url: 'check-bc-logout',
```
Comment (brionmario) [inline sso-samples/oidc-sso-sample/pickup-manager/src/main/webapp/bcIFrame.jsp:10]: Shall we use double quotes. Check other places as well.

```suggestion
            url: "check-bc-logout",
```

Comment (Kayathiri-Mahendrakumaran) [reply sso-samples/oidc-sso-sample/pickup-manager/src/main/webapp/bcIFrame.jsp]: Addressed with abacc0a820911a6e100fef13d6845ae9845ac813

Comment (Kayathiri-Mahendrakumaran) [reply sso-samples/oidc-sso-sample/pickup-manager/src/main/webapp/bcIFrame.jsp]: Addressed with abacc0a820911a6e100fef13d6845ae9845ac813

**Review by ashendes — APPROVED** [review]

**Pull request merged**